src/assimp/assimp.cpp

```diff
@@ -506,7 +508,13 @@ assimp::Implementation::BindState assimp::Implementation::processMaterials(const
             material->Get(AI_MATKEY_COLOR_EMISSIVE, pbr.emissiveFactor);
             material->Get(AI_MATKEY_GLTF_ALPHACUTOFF, pbr.alphaMaskCutoff);
 
-            if (material->Get(AI_MATKEY_TWOSIDED, isTwoSided); isTwoSided)
+            bool optionFlag{false};
+            if(options->getValue(assimp::two_sided, optionFlag) && optionFlag) 
```

Comment:
A further refinement would be to replace the above two lines with:

    if (vsg::value<bool>(false, assimp::two_sided, options))

Though, I'm not sure the code is actually more clear...  I'll merged as is then try streamlining the code.